guide/moderator/sticky.md

```diff
@@ -0,0 +1,16 @@
+## User Handbook
+
+### Sticky/Unsticky a thread
```

Comment:
Let's note here what Sticky does (and that the extension must be enabled to have this feature). Sticky posts rise to the top of the All Discussions list until a user has read all new posts within. They will always live at the top of their respective tag's discussion list.